leap_year.rb

```diff
@@ -0,0 +1,38 @@
+def leap_years
+    @leap_years_list = []
+    puts "Enter a starting year here"
+    @starting_year = gets.chomp
+    puts "Now enter an ending year"
+    @ending_year = gets.chomp
+    for year in @starting_year.to_i..@ending_year.to_i do    
+        if year % 4 == 0
+            if year % 100 == 0 && year % 400 == 0
+            # A leap year is divisible by 4,
+            # except for years that are divisible by 100 -- those aren't leap years
+            # except for years that are divisible by 400, which ARE leap years.
+                @leap_years_list.push(year)
+            end
+            if year % 100 != 0
+                @leap_years_list.push(year)
+            end      
```

Comment:
This extra if statement is redundant, you could just add this extra condition to the previous if statement and avoid extra lines of code 😄 
```suggestion
            if year % 100 == 0 && year % 400 == 0 || year % 100 != 0
            # A leap year is divisible by 4,
            # except for years that are divisible by 100 -- those aren't leap years
            # except for years that are divisible by 400, which ARE leap years.
                @leap_years_list.push(year)
            end
```